src/guides/ui-elements/index.md

```diff
@@ -0,0 +1,234 @@
+---
+uid: ui-elements
+---
+
+# User Interface Elements
+
+Abbot supports a limited set of UI elements that can be leveraged in a skill. Unfortunately, these elements only work for Slack and Teams. Discord doesn't yet support UI elements. We may have a workaround for that in the future, but for now, this is the way it is.
+
+Over time, Abbot will add support for more and more elements.
+
+## Buttons
+
+Buttons are a simple way to present the user a set of choices. They can be useful in workflows driven by chat such as an approval workflow. For example, suppose a skill needs to find out the user's favorite color. The following code will reply with some text and a set of buttons.
+
+# [C#](#tab/tabid-cs)
+
+```cs
+var buttons = new[] {
+    new Button("Red"),
+    new Button("Blue"),
+    new Button("Green")
+};
+await Bot.ReplyWithButtonsAsync("Please choose a color.", buttons);
+```
+
+# [JavaScript](#tab/tabid-js)
+
+```javascript
+module.exports = (async () => { // We modularize your code in order to run it.
```

Comment:
If the user copies and pastes this example directly into a new JavaScript skill, I think they will get errors. We add the first and last line in the editor to make it clearer how we run their code, and so that jslint doesn't freak out when they try to do async calls outside of a function.

Not sure if we want to include those lines in the examples or not.

Reply:
Yeah, we should omit them. Especially if we come up with a solution down the road where we don't need to show those lines to the user in the editor.